packages/jbrowse-cli/src/testUtil.ts

```diff
@@ -74,16 +69,17 @@ export const test = oclifTest
 export const setup = test
   .mockStdoutWrite()
   .add('originalDir', () => process.cwd())
-  .add('dir', async () => {
+  .add('dir', () => {
+    const tmpDir = fs.realpathSync(os.tmpdir())
     const jbrowseTmpDir = path.join(tmpDir, 'jbrowse')
-    await fsPromises.mkdir(jbrowseTmpDir, { recursive: true })
-    return fsPromises.mkdtemp(path.join(jbrowseTmpDir, path.sep))
+    fs.mkdirSync(jbrowseTmpDir, { recursive: true })
+    return fs.mkdtempSync(path.join(jbrowseTmpDir, path.sep))
```

Comment:
Why were these made synchronous?

Reply:
not sure I see the advantage of fs.promises, so I changed them to sync. I reverted though

Reply:
to be more clear I thought there was something weird happening where tests were running into each other, and I thought I needed to await the results in the test itself e.g. await setup. this seems not true, and I was probably mixed up, but having the proper promise for the unzip completion fixed tests

Reply:
I have a theory of why the promise fixed the tests, I was running into the same thing before. The oclif `command` test util probably `await`s the command, but prior to this change the command was still writing streams when it returned, causing race conditions. In normal CLI usage this wouldn't happen since the node process wouldn't exit until all the streams were written, which I"m guessing is why I couldn't reproduce it outside of tests.

I think with your fix you can probably remove the `sleep`s I put in create.test.ts, which I put in to fix this problem before.

Reply:
Simply removing the sleeps seems to not work. I am pretty confident in the new Promise concept here because it makes sure that a promise is returned from the async run for unzip completion

Reply:
Ok, I must have been actually running into a different problem, then. Yeah, I think your promise is the right way to go.

Reply:
I found maybe a solution to allow removing the sleeps, and improve the promise that is returned, there is a .promise() helper on the unzip stream https://github.com/ZJONSSON/node-unzipper#parsepromise-syntax-sugar